usaspending_api/etl/tests/test_elasticsearch_indexer.py

```diff
@@ -0,0 +1,368 @@
+        is_fpds=True,
+        type="A",
+        transaction_unique_id=fpds_unique_key,
+    )
+    mommy.make(
+        "awards.TransactionFPDS",
+        detached_award_proc_unique=fpds_unique_key,
+        transaction_id=1,
+        legal_entity_zip5="abcde",
+        piid="IND12PB00323",
+        legal_entity_county_code="059",
+        legal_entity_state_code="VA",
+        legal_entity_congressional="11",
+        legal_entity_country_code="USA",
+        place_of_performance_state="VA",
+        place_of_performance_congr="11",
+        place_of_perform_country_c="USA",
+        naics="331122",
+        product_or_service_code="1510",
+        type_set_aside="8AN",
+        type_of_contract_pricing="2",
+        extent_competed="F",
+    )
+
+    fabs_unique_key = "fabs_transaction_id_2"
+    mommy.make(
+        "awards.TransactionNormalized",
+        id=2,
+        award_id=2,
+        action_date="2016-10-01",
+        is_fpds=False,
+        type="02",
+        transaction_unique_id=fabs_unique_key,
+    )
+    mommy.make(
+        "awards.TransactionFABS",
+        transaction_id=2,
+        fain="P063P100612",
+        cfda_number="84.063",
+        afa_generated_unique=fabs_unique_key,
+    )
+
+    mommy.make("references.ToptierAgency", toptier_agency_id=1, name="Department of Transportation")
+    mommy.make("references.SubtierAgency", subtier_agency_id=1, name="Department of Transportation")
+    mommy.make("references.Agency", id=1, toptier_agency_id=1, subtier_agency_id=1)
+    mommy.make(
+        "awards.Award",
+        id=1,
+        generated_unique_award_id="CONT_AWD_IND12PB00323",
+        latest_transaction_id=1,
+        is_fpds=True,
+        type="A",
+        piid="IND12PB00323",
+        description="pop tarts and assorted cereals",
+        total_obligation=500000.00,
+        date_signed="2010-10-1",
+        awarding_agency_id=1,
+        funding_agency_id=1,
+        update_date="2012-05-19",
+    )
+    mommy.make(
+        "awards.Award",
+        id=2,
+        generated_unique_award_id="ASST_NON_P063P100612",
+        latest_transaction_id=2,
+        is_fpds=False,
+        type="02",
+        fain="P063P100612",
+        total_obligation=1000000.00,
+        date_signed="2016-10-1",
+        update_date="2014-07-21",
+    )
+    mommy.make("accounts.FederalAccount", id=1)
+    mommy.make(
+        "accounts.TreasuryAppropriationAccount",
+        treasury_account_identifier=1,
+        agency_id="097",
+        main_account_code="4930",
+        federal_account_id=1,
+    )
+
+    mommy.make("awards.FinancialAccountsByAwards", financial_accounts_by_awards_id=1, award_id=1, treasury_account_id=1)
+
+
+award_config = {
+    "create_new_index": True,
+    "data_type": "award",
+    "data_transform_func": transform_award_data,
+    "directory": Path(__file__).resolve().parent,
+    "fiscal_years": [2008, 2009, 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020],
```

Comment:
I think `directory` and `fiscal_years` are obsolete. `skip_counts` looks obsolete too even though it lingers in one tuple in elasticsearch_indexer.py

Reply:
K. I don't think i changed those, so if they show as changed, it may have been black formatting. I'll create a tech debt ticket to remove obsolete args.

Reply:
https://federal-spending-transparency.atlassian.net/browse/DEV-6776